src/angular/file-selector/file-selector.ts

```diff
@@ -232,9 +232,4 @@ export class SbbFileSelector
       file1.lastModified === file2.lastModified
     );
   }
-
-  static ngAcceptInputType_capture: 'user' | 'environment' | string | null | undefined;
```

Comment:
Here I am also not that sure, because types here are not the same as they are here (string missing): 
```ts
/** Attribute whose state specifies the preferred facing mode for the media capture mechanism. */
  @Input() capture?: 'user' | 'environment';
```